### PR TITLE
Waste theme

### PR DESCRIPTION
**Pull request by RobinLinde**
Waste theme should be almost complete now, a few points/issues which I'm not 100% sure about:

- [ ] Waste baskets for trash and plastic or trash and paper (e.g. [image1](https://services1.arcgis.com/p5QhXC0i0sZjprM1/arcgis/rest/services/Dataset_Raadplegen_Afvalbakken_Assen/FeatureServer/0/636/attachments/620) and [image 2](https://services1.arcgis.com/p5QhXC0i0sZjprM1/arcgis/rest/services/Dataset_Raadplegen_Afvalbakken_Assen/FeatureServer/0/526/attachments/510))
- [x] Clear distinction between `amenity=waste_disposal` and `amenity=recycling` with `recycling:waste` (allow changing)

Preview available here: https://pietervdvn.github.io/mc/waste-theme/waste.html
Any other feedback is also very welcome.

### CODE REVIEW

File: assets/layers/recycling/recycling.json [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+      ]
+    },
+    {
+      "id": "recycling-accepts",
+      "question": {
+        "en": "What can be recycled here?",
+        "nl": "Wat kan hier gerecycled worden?"
+      },
+      "multiAnswer": true,
+      "mappings": [
```
Comment (pietervdvn) [inline assets/layers/recycling/recycling.json:204]: I'm missing 'organic waste' a bit here, that would be a useful addition.

File: assets/themes/waste/waste.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "icon": "./assets/layers/recycling/recycling-14.svg",
+  "startZoom": 19,
+  "startLat": 53.24865,
+  "startLon": 6.60075,
+  "layers": [
+    {
+      "builtin": "waste_basket",
+      "override": {
+        "minzoom": 12
```
Comment (pietervdvn) [inline assets/themes/waste/waste.json:25]: I would hide the waste basket at lower zoom levels, and only opt to show them at >16. Right now, they clutter the map.

As user of the 'waste'-map, I'm interested in where I can drop my glass/metal/green_waste on city scale. If I'm looking for a waste basket, I'd have zoomed in a whole lot more to a waste basket in my vicinity.

File: assets/themes/waste/waste.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      }
+    },
+    "recycling",
+    "waste_disposal"
```
Comment (pietervdvn) [inline assets/themes/waste/waste.json:29]: Waste disposal is a bit of a special case, but it is worth considering setting a higher minzoom too as they won't be useful for someone glancing over the map.

**Comment by pietervdvn** [comment]
> * Waste baskets for trash and plastic or trash and paper (e.g. [image1](https://services1.arcgis.com/p5QhXC0i0sZjprM1/arcgis/rest/services/Dataset_Raadplegen_Afvalbakken_Assen/FeatureServer/0/636/attachments/620) and [image 2](https://services1.arcgis.com/p5QhXC0i0sZjprM1/arcgis/rest/services/Dataset_Raadplegen_Afvalbakken_Assen/FeatureServer/0/526/attachments/510))

For these, I'd simply mark that it takes both of these types of trash. Once could add `segregated=yes` to indicate that the items are separated.





> * Clear distinction between `amenity=waste_disposal` and `amenity=recycling` with `recycling:waste` (allow changing)

This is a tricky one. Maybe we should rethink this distinction and see 'amenity=waste_disposal' and 'amenity=recycling' as one attribute to the same layer - thus effectively merge them, making it easy to switch between the types.

File: assets/layers/recycling/recycling.json [hunk 4]
```diff
@@ -0,0 +1,532 @@
...
+      }
+    ]
+  },
+  "mapRendering": [
+    {
+      "location": [
+        "point",
+        "centroid"
+      ],
+      "icon": "./assets/layers/recycling/recycling-14.svg",
```
Comment (pietervdvn) [inline assets/layers/recycling/recycling.json:43]: Perhaps use `"icon":"circle:white;./assets/layers/recycling/recycling-14.svg"` to add a white circle backfill. This prevents weird rendering when two points overlap: 
image

And it prevents the background layer to 'bleed through': 
image

File: assets/layers/recycling/recycling.json (hunk 1 above)
Comment (RobinLinde) [reply assets/layers/recycling/recycling.json:204]: Currently it does include `recycling:green_waste`, which feels very similar, and seems to be used more

File: assets/themes/waste/waste.json (hunk 2 above)
Comment (RobinLinde) [reply assets/themes/waste/waste.json]: Good point, minzoom is now set to 18, feels more reasonable for me

File: assets/themes/waste/waste.json (hunk 3 above)
Comment (RobinLinde) [reply assets/themes/waste/waste.json:30]: Minzoom now also 18, still thinking about merging with recycling (need some more sleep first 😉)

File: assets/themes/waste/waste.json (hunk 2 above)
Comment (pietervdvn) [reply assets/themes/waste/waste.json]: Yes, way better!

File: assets/themes/waste/waste.json (hunk 3 above)
Comment (pietervdvn) [reply assets/themes/waste/waste.json:30]: Haha, yeah, good night!

File: assets/layers/recycling/recycling.json [hunk 5]
```diff
@@ -40,89 +43,355 @@
         "point",
         "centroid"
       ],
-      "icon": "circle:white;./assets/layers/recycling/recycling-14.svg",
+      "icon": {
```
Comment (pietervdvn) [inline assets/layers/recycling/recycling.json:46]: Oooh, wow, clever! And still a somewhat manageable construction...

**Comment by pietervdvn** [comment]
IMHO, this PR is ready to roll! @RobinLinde Any more blocking issues? If not, I'm gonna merge it in.

**Review by pietervdvn — APPROVED** [review]

**Pull request merged**